gf-webapp-admin/src/component/sideMenu/addRoomModal/AddRoomModal.tsx

```diff
@@ -15,21 +15,24 @@ import {Services} from "src/component/sideMenu/addRoomModal/services/Services";
 import {Square} from "src/component/sideMenu/addRoomModal/square/Square";
 import {Button} from "gf-ui-lib/src/components/Button/Button";
 import {postRoom} from "src/service/RoomService";
+import {NewRoom} from "src/model/Room/NewRoom";
 import styles from "src/component/sideMenu/addRoomModal/AddRoomModal.module.scss";
 
 interface ShowModalProps {
   showModal: () => void
 }
 
 export const AddRoomModal = (props: ShowModalProps) => {
-  const {roomStartState} = useContext(AddRoomContext);
+
+  const [room, setRoom] = useState<NewRoom>(defaultRoom);
+
   const addRoom = async () => {
-    await postRoom(roomStartState);
+    await postRoom(room);
     location.reload();
```

Comment:
o-o. Bad decision for react) do u know why? Try to guess and answer please)

Reply:
If the room is null, then the page will not go to reload?